### ISSUE TITLE
Add information on the lock voting

### ISSUE THREAD

**Issue by lsaether** [w3f/polkadot-wiki-old#148]
We have some information on Governance but I think we should put some effort into expanding on some of the topics. Especially lock voting comes to mind as something that should be expanded.. Why is it better than just plain coin voting? What other mechanisms are out there for voting?

Compare the recent Aragon vote, where the governance was based on simple coin vote and it became [governance by one whale](https://www.evanvanness.com/post/184616403861/aragon-vote-shows-the-perils-of-onchain-governance). How does lock vote make whales increase their exposure to their choice?

**Comment by ltfschoen** [comment]
I recall Gav mentioning this at Sub0, which was recorded and should be published on youtube

**Comment by ansonla3** [comment]
should #169 answer some of them?

**Comment by lsaether** [comment]
Yep, but I also want to include the case study I posted above to better illustrate why coin voting in general is hard and why mechanisms like lock voting can mitigate some attacks.
